extensions/ipynb/extension.webpack.config.js

```diff
@@ -8,13 +8,20 @@
 'use strict';
 
 const withDefaults = require('../shared.webpack.config');
+const path = require('path');
 
 module.exports = withDefaults({
 	context: __dirname,
 	entry: {
-		extension: './src/ipynbMain.ts',
+		ipynbMain: './src/ipynbMain.ts',
```

Comment:
@rebornix I do not think this will work,
We will need to ensure we have 2 builds, one for browser and one for web,
trying to import `worker_threads` when building for web could fall over in build, thats what I think.